hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -168,6 +168,8 @@
     @Deprecated
     public static final String PROP_ENABLE_JMX_DETAILED = GroupProperty.ENABLE_JMX_DETAILED.getName();
     @Deprecated
+    public static final String PROP_JMX_UPDATE_INTERVAL_SECONDS = GroupProperty.JMX_UPDATE_INTERVAL_SECONDS.getName();
```

Comment:
Yes, please don't add anything to `GroupProperties` anymore. This is the old way and a grey-zone between public and private API. Just put stuff in `GroupProperty`, which is new since 3.6 and clearly marked as `@PrivateAPI`, so we can change things as we need to in the future.